rfc6962-bis.xml

```diff
@@ -752,6 +752,17 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
           <spanx style="verb">inclusion_path</spanx> is a vector of Merkle Tree nodes proving the inclusion of the chosen certificate or precertificate.
         </t>
       </section>
+      <section title="Shutting down a log">
+        <t>
+          Log operators may decide to shut down a log for various reasons, such as depracation of signature algorithm. If there are entries in the log for certificates that have not yet expired, simply making TLS clients stop recognizing that log will have the effect of invalidating SCTs from that log. To avoid that, the following actions are suggested:
+          <list style="symbols">
+            <t>Make it known to clients and monitors that the log will be frozen.</t>
+            <t>Stop accepting new submissions (the error code shutdown should be returned for such requests).</t>
+            <t>Once MMD from the last accepted submission has passed and all pending submissions are incorporated, issue a final STH and publish it as a part of the log's metadata. At this point the log's private key is no longer needed and can be destroyed.</t>
```

Comment:
That should guarantee that no client sees an SCT where (timestamp + MMD) > timestamp in final STH.
This is an attempt to simplify client implementation by not requiring it to be aware of the particular log shutdown case: a client can simply get the latest STH, find out that timestamp from SCT + MMD has passed and request an inclusion proof like any other case.